azurerm/internal/services/web/function_app.go

```diff
@@ -131,6 +131,26 @@ func schemaAppServiceFunctionAppSiteConfig() *pluginsdk.Schema {
 					Optional:     true,
 					ValidateFunc: validation.StringInSlice([]string{"1.8", "11"}, false),
 				},
+
+				"minimum_elastic_instance_count": {
+					Type:         pluginsdk.TypeInt,
+					Optional:     true,
+					Computed:     true,
+					ValidateFunc: validation.IntBetween(0, 20),
+				},
+
+				"function_app_scale_limit": {
```

Comment:
function app is in the resource name so
```suggestion
				"scale_limit": {
```
also does this have a unit?

Reply:
It seems easier to understand include `app` since it sets the scale-out limit for each application. like `app_scale_limit`

Reply:
sounds good to me!